### DISCUSSION TITLE
Conversations within Pull Requests, not nice to work with

### DISCUSSION THREAD

**Discussion by axel-meyer-fnt** [community/community#108302] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Hello everyone,

I have a question/feedback regarding conversations (comments in code) within Pull requests.

I would like to know, how one is supposed to effectively work with them.

I'm working within a bigger project, and am supposed to do a lot of code review there.
So everyday I comment on several commits, and either request changes or ask some question there.

But how am I supposed to keep track of such conversations?

I need some way to get notified if someone answers or closes a conversation that I opened up.
But I don't see such options anywhere.

If i generally have notifications activated for pull requests that I'm involved in, I get several 100s emails a day, which is not useful.

If I want to check conversations afterwards I have to dig to through a lot of pull requests, and open up all conversations to check if the requested changes were really done correctly or to find if someone answered my question.

I found similar topic:
https://github.com/orgs/community/discussions/7638
But it also does not help much, since we have some uncommon requirements in the project (don't ask) where we only have one commit in a PR and force push afterwards. So conversations can't be found that way again, because they seem to be related to a single commit.

Surly others have this problem too, so how can one deal with this problem?

Greetings
Axel

**Comment by kyuna0312** [comment]
it's help for learning and know about what kind of code you write in before and if you get any error it should help to back your working code

**Reply by kyuna0312** [reply]
you should write what kind of things you did and it will help for your code and improvement

**Reply by kyuna0312** [reply]
and then make this as answer thank you :3

**Comment by axel-meyer-fnt** [comment]
So mostly this was a misunderstanding in the options of notifications.

[ ] "Pull Request Reviews"
[x] "Comments On Issues and Pull Requests"

I only had activated second option, because I was assuming, that reviews in this case only meant being notified about someone approving or declining a PR.
